image/bin/createusers.py

```diff
@@ -0,0 +1,30 @@
+"""
+Create 3 users in the db:
+* user: a regular user
+* developer: a developer (triager) that signed the cla
+* coordinator: a coordinator and committer that signed the cla
+
+Most of the fields are the same as the username.
+The password for all the users is 'pass'.
+"""
+
+import subprocess
+
+rdadmin = ['/home/tracker/bin/roundup-admin', '-i', '/opt/tracker/python-dev']
+
+users = ['user', 'developer', 'coordinator']
+roles = []
+
+for user in users:
+    roles.append(user.capitalize())
+    r = ','.join(roles)
```

Comment:
It would be nice adding a comment here, that developer has at each point user needs to have his role and all previous ones.